exposure-notification/src/main/java/fi/thl/covid19/exposurenotification/efgs/FederationGatewaySyncProcessor.java

```diff
@@ -33,7 +35,7 @@ private void runExportToEfgs() {
             fixedRateString = "${covid19.federation-gateway.download-interval}")
     private void runImportFromEfgs() {
         LOG.info("Starting scheduled import from efgs.");
```

Comment:
Might be useful to include in the logs some identifying information for the operation - the date/operation id, etc.

Reply:
There is a timestamp for log row, not sure what else we could have in here, because this is scheduled operation.

Reply:
I was thinking about the date that is given into the startInbound-call (you're right, it's already in the timestamp, so kind of redundant) and maybe the operation id on the finished-log-row (if startInbound returns it). Perhaps it's already obvious from the other rows that get logged during the op.

Reply:
Yep, currently there is no id returned, but it might be indeed a good addition.  Good point. Let's see after initial testing.